src/earn/poolInfoScreen/EarnPoolInfoScreen.tsx

```diff
@@ -209,10 +218,10 @@ export default function EarnPoolInfoScreen({ route, navigation }: Props) {
     const partialWithdrawalsEnabled = getFeatureGate(
       StatsigFeatureGates.ALLOW_EARN_PARTIAL_WITHDRAWAL
     )
-    if (partialWithdrawalsEnabled) {
-      navigate(Screens.EarnEnterAmount, { pool, mode: 'withdraw' })
+    if (hasRewards || partialWithdrawalsEnabled) {
+      withdrawBottomSheetRef.current?.snapToIndex(0)
     } else {
-      navigate(Screens.EarnCollectScreen, { pool })
+      navigate(Screens.EarnCollectScreen, { pool }) // TODO (ACT-1389): Confirmation screen for Claim & Withdraw
```

Comment:
question on the todo here. should we keep the gate off flow the current one? i.e., all changes for partial withdrawals  / new screens go behind the gate, cc @MuckT

Reply:
We should fix this so the old flow is served by default until the feature gate is enabled.

Reply:
@satish-ravi @MuckT are you suggesting that we use `ALLOW_EARN_PARTIAL_WITHDRAWAL` as the gate to decide whether the new flow is shown, or adding an additional `SHOW_EARN_WITHDRAW_OPTIONS` gate on top of the existing one?